Detections/MultipleDataSources/ThalliumIOCs.yaml

```diff
@@ -0,0 +1,53 @@
+id: 70b12a3b-4896-42cb-910c-5ffaf8d7987d
+name: THALLIUM domains included in DCU takedown
+description: |
+  'THALLIUM spearphishing and command and control domains included in December 2019 DCU/MSTIC takedown. 
+   Matches domain name IOCs related to the THALLIUM activity group with CommonSecurityLog, DnsEvents, VMConnection and SecurityEvents dataTypes.
+   References: https://blogs.microsoft.com/on-the-issues/2019/12/30/microsoft-court-action-against-nation-state-cybercrime/ '
+severity: High
+requiredDataConnectors:
+  - connectorId: DNS
+    dataTypes:
+      - DnsEvents
+  - connectorId: AzureMonitor(VMInsights) 
+    dataTypes:
+      - VMConnection
+  - connectorId: CiscoASA
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: PaloAltoNetworks
+    dataTypes:
+      - CommonSecurityLog
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - CommandAndControl
+  - CredentialAccess
+query: |
+  let timeframe = 1d;
+  let DomainNames = dynamic(["seoulhobi.biz", reader.cash", pieceview.club", app-wallet.com", bigwnet.com", bitwoll.com", cexrout.com", change-pw.com", checkprofie.com", cloudwebappservice.com", ctquast.com", dataviewering.com", day-post.com", dialy-post.com", documentviewingcom.com", dovvn-mail.com", down-error.com", drivecheckingcom.com", drog-service.com", encodingmail.com", filinvestment.com", foldershareing.com", golangapis.com", hotrnall.com", lh-logins.com", login-use.com", mail-down.com", matmiho.com", mihomat.com", natwpersonal-online.com", nidlogin.com", nid-login.com", nidlogon.com", pw-change.com", rnaii.com", rnailm.com", sec-live.com", secrityprocessing.com", securitedmode.com", securytingmail.com", set-login.com", usrchecking.com", com-serviceround.info", mai1.info", reviewer.mobi", files-download.net", fixcool.net", hanrnaii.net", office356-us.org", smtper.org"]);
+  (union isfuzzy=true
+  (CommonSecurityLog 
+  | where TimeGenerated >= ago(timeframe) 
+  | parse Message with * '(' DNSName ')' * 
+  | where isnotempty(FileHash)
+  | where DNSName in~ (DomainNames)
+  | extend Account = SourceUserID, Computer = DeviceName, IPAddress = SourceIP
+  ),
+  (DnsEvents 
+  | where TimeGenerated >= ago(timeframe) 
+  | where isnotempty(DNSName)
+  | where DNSName  in~ (DomainNames)
```

Comment:
Wondering if DNSName is always just the second level domain or is it sometimes more and we should use has_any versus in~